scripts/umbrel-os/external-storage/mount

```diff
@@ -200,6 +200,22 @@ main () {
   mkswap "${SWAP_FILE}"
   swapon "${SWAP_FILE}"
 
+  echo "Checking if Docker data dir is on external storage..."
+  if [[ -f "${MOUNT_POINT}/docker" ]]; then
+    echo "Yes, Docker data dir is on the SSD"
+  else 
+    echo "No, Docker data dir is on the SD card"
+    echo "Moving data..."
+    cp  --recursive \
+        --archive \
+        --no-target-directory \
+        /var/lib/docker ${MOUNT_POINT}/docker
+  fi
+  mount --bind ${MOUNT_POINT}/docker /var/lib/docker
```

Comment:
You should always quote variables to prevent string splitting in bash.

```suggestion
  mount --bind "${MOUNT_POINT}/docker" /var/lib/docker
```

For example if `MOUNT_POINT` was `/mnt/Samsung T5` then you'd end up running

```
mount --bind /mnt/Samsung T5/docker /var/lib/docker
```

instead of

```
mount --bind "/mnt/Samsung T5/docker" /var/lib/docker
```